frontend/pages/software/ManageSoftwarePage/SoftwareTableConfig.tsx

```diff
@@ -60,53 +61,56 @@ const condenseVulnerabilities = (
     : condensed;
 };
 
-const softwareTableHeaders = [
-  {
-    title: "Name",
-    Header: "Name",
-    disableSortBy: true,
-    accessor: "name",
-    Cell: (cellProps: IStringCellProps): JSX.Element => {
-      const { name, bundle_identifier } = cellProps.row.original;
-      if (bundle_identifier) {
-        return (
-          <span className="name-container">
-            <TooltipWrapper
-              tipContent={`
-                <span>
-                  <b>Bundle identifier: </b>
-                  <br />
-                  ${bundle_identifier}
-                </span>
-              `}
-            >
-              {name}
-            </TooltipWrapper>
-          </span>
-        );
-      }
-      return <TextCell value={name} />;
+const getMaxProbability = (vulns: IVulnerability[]) =>
+  vulns.reduce((maxProb, { epss_probability: prob }) => {
+    return prob && prob > maxProb ? prob : maxProb;
+  }, 0);
```

Comment:
a potentially slightly clear version of this:

```suggestion
  vulns.reduce((max, { epss_probability }) => Math.max(max, epss_probability || 0), 0);
```